values.sample.yaml

```diff
@@ -140,6 +140,20 @@ tonicai:
 # new one created.  This will result in Tonic being down during an update.
 deployStrategy: RollingUpdate
 
+containerization:
+  # if self managing RBAC, to use containerization the tonic service account
+  # will need a rolebinding that grants:
+  # rules:
+  #   - apiGroups: [""]
+  #     resources: ["pods"]
+  #     verbs: ["get", "list", "delete", "create", "watch", "deletecollection"]
+  #   - apiGroups: [""]
+  #     resources: ["secrets"]
+  #     verbs: ["list", "delete", "create", "deletecollection"]
+  rbac:
+    create: true
+  datapacker:
+    imageRepo: quay.io/tonicai/datapacker
```

Comment:
Should this live under a key? A features key here or the one on tonic web? Keys are hard to rehome after landing in a public version.